das/checkpoint_store.go

```diff
@@ -0,0 +1,45 @@
+package das
+
+import (
+	"encoding/binary"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+)
+
+var (
+	storePrefix   = datastore.NewKey("das")
+	checkpointKey = datastore.NewKey("checkpoint")
+)
+
+// NewCheckpointStore wraps the given datastore.Datastore with
+// the `das` prefix.
+func NewCheckpointStore(ds datastore.Datastore) datastore.Datastore {
```

Comment:
Hmmm. Is this it best name for this? It's not really making a _new_ store if it wraps a given store.

Reply:
renamed to `wrapCheckpointStore`

Reply:
Fixed in [ee95275](https://github.com/celestiaorg/celestia-node/pull/473/commits/ee952758c5c23f631937be7326e069d8c834721b)